extensions/oidc-client/runtime/src/main/java/io/quarkus/oidc/client/runtime/OidcClientRecorder.java

```diff
@@ -110,34 +109,32 @@ protected static OidcClient createOidcClient(OidcClientConfig oidcConfig, String
         try {
             OidcCommonUtils.verifyCommonConfiguration(oidcConfig, false);
         } catch (Throwable t) {
+            LOG.debug(t.getMessage());
```

Comment:
Would make more sense to log the stack trace?

Reply:
@pedroigor, probably not - the message would say something like `auth-server-url must be set, etc` so the stacktrace would just show the flow via Oidc(Client)Recorder to `OidcCommonUtils` - I believe the stacktraces are not logged in general in Quarkus